src/lib/make-toolbox-xml.js

```diff
@@ -354,45 +373,55 @@ const control = function () {
         ${blockSeparator}
         <block type="control_stop"/>
         ${blockSeparator}
-        <block type="control_start_as_clone"/>
-        <block type="control_create_clone_of">
-            <value name="CLONE_OPTION">
-                <shadow type="control_create_clone_of_menu"/>
-            </value>
-        </block>
-        <block type="control_delete_this_clone"/>
+        ${isStage ? `
+            <block type="control_create_clone_of">
+                <value name="CLONE_OPTION">
+                    <shadow type="control_create_clone_of_menu"/>
```

Comment:
Maybe for a different PR but this still has the 'myself' option

Reply:
Hm, well fixed it since it is kinda related: 8aa1d6e